lib/health_check/middleware_health_check.rb

```diff
@@ -7,16 +7,15 @@ def initialize(app)
 
     def call(env)
       uri = env['PATH_INFO']
-      if uri =~ /^\/?#{HealthCheck.uri}\/?([-_0-9a-zA-Z]*)([-_0-9a-zA-Z]*)(\.(\w*))?/
```

Comment:
I don't understand why it was ([-_0-9a-zA-Z]*)([-_0-9a-zA-Z]*)
instead of middleware_([-_0-9a-zA-Z]*) 
also checks list evaluation is to complicated